runtime/jcl/CMakeLists.txt

```diff
@@ -157,6 +157,7 @@ target_sources(jclse
 		${CMAKE_CURRENT_SOURCE_DIR}/common/system.c
 		${CMAKE_CURRENT_SOURCE_DIR}/common/thread.cpp
 		${CMAKE_CURRENT_SOURCE_DIR}/common/unsafe_mem.c
+		${CMAKE_CURRENT_SOURCE_DIR}/common/vectornatives.cpp
```

Comment:
Actually, this might need to be only included on jdk16+.  Maybe use similar code to the jdk9+ example below in the file:
```
# We'd like to use GREATER_EQUAL here (and below), but that operator requires
# cmake version 3.7 or better; the minimum version we require is 3.4.	# cmake version 3.7 or better; the minimum version we require is 3.4.
if(NOT JAVA_SPEC_VERSION LESS 16)
	# sources for Java 16+
	target_sources(jclse
		PRIVATE
			${CMAKE_CURRENT_SOURCE_DIR}/common/java_lang_invoke_VarHandle.c
			${CMAKE_CURRENT_SOURCE_DIR}/common/java_lang_StackWalker.cpp
	)
endif()
```

@dnakamura do you have a better suggestion for how to handle a jdk16+ only source file?

Reply:
no, that would be the way to do it.